### PR TITLE
course correction bug fix + logging erogonmics

### PR DESCRIPTION
**Pull request by nfranczak**

### CODE REVIEW

File: components/base/kinematicbase/execution.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ const (
 
 	// Used to determine minimum linear deviation allowed before correction attempt. Determined by multiplying max linear speed by
 	// inputUpdateStepSeconds, and will correct if deviation is larger than this percent of that amount.
-	minDeviationToCorrectPct = 50.
+	minDeviationToCorrectPct = 70.
```
Comment (biotinker) [inline components/base/kinematicbase/execution.go:32]: I don't think we want to change this as part of this PR; let's just do the logs + the bugfix.

File: components/base/kinematicbase/execution.go [hunk 2]
```diff
@@ -331,9 +331,9 @@ func (ptgk *ptgBaseKinematics) courseCorrect(
 		"allowable diff ", allowableDiff, " diff now ", poseDiff.Point().Norm(), " angle diff ", poseDiff.Orientation().AxisAngles().Theta,
 	)
 
+	ptgk.logger.Debug("expected to be at ", spatialmath.PoseToProtobuf(expectedPose))
+	ptgk.logger.Debug("Localizer says at ", spatialmath.PoseToProtobuf(actualPose.Pose()))
```
Comment (biotinker) [inline components/base/kinematicbase/execution.go:335]: I don't think we want this log change to be permanent, it would be super chatty.

**Review by biotinker — CHANGES_REQUESTED** [review]
Couple small tweaks but otherwise looks good.

File: components/base/kinematicbase/execution.go (hunk 2 above)
Comment (nfranczak) [reply components/base/kinematicbase/execution.go:335]: sounds good, I will revert.

File: components/base/kinematicbase/execution.go [hunk 3]
```diff
@@ -49,11 +49,11 @@ type arcStep struct {
 }
 
 func (step *arcStep) String() string {
-	return fmt.Sprintf("Step: lin velocity: %f, ang velocity: %f, duration: %f s, arcSegment %+v, arc start pose %s",
+	return fmt.Sprintf("Step: lin velocity: %f,\n\t ang velocity: %f,\n\t duration: %f s,\n\t arcSegment %s,\n\t arc start pose %v",
 		step.linVelMMps,
 		step.angVelDegps,
 		step.durationSeconds,
-		step.arcSegment,
+		step.arcSegment.String(),
```
Comment (raybjork) [inline components/base/kinematicbase/execution.go:56]: I think if you are using the `%s` formatting directive this is implied and therefore unnecessary

Comment (nfranczak) [reply components/base/kinematicbase/execution.go:56]: ooo, I'll double check this on my end and update.

**Review by biotinker — APPROVED** [review]
Lgtm

File: components/base/kinematicbase/execution.go (hunk 3 above)
Comment (nfranczak) [reply components/base/kinematicbase/execution.go:56]: i think we need it "fmt.Sprintf format %s has arg step.arcSegment of wrong type go.viam.com/rdk/motionplan/ik.Segment"

**Pull request merged**